go/ldflags-darwin.go

```diff
@@ -0,0 +1,14 @@
+//go:build darwin
+
+/*
+ * Copyright (C) Danielle De Leo
+ * Copyright (C) NGINX, Inc.
+ */
+
+package unit
+
+/*
+#cgo LDFLAGS: -L/opt/homebrew/include -L/opt/homebrew/lib
```

Comment:
Hmm, is `-L/opt/homebrew/include` really needed here? I hope there's no libraries in there!

Reply:
Good point :) Thanks Andrew!